Projects/Features/StorageFeature/Sources/ViewControllers/LikeStorageViewController.swift

```diff
@@ -40,6 +40,11 @@ final class LikeStorageViewController: BaseReactorViewController<LikeStorageReac
         setTableView()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated: animated)
```

Comment:
```suggestion
        super.viewDidAppear(animated)
```
어 이거 수정전으로 suggestion 남은거같은데 unnamed label이라 animated붙이면 아마 안될거에요 🥺